docs/guides/guide-vmcluster-multiple-retention-setup/README.md

```diff
@@ -48,3 +57,12 @@ Every group of vmstorages can handle one tenant or multiple one. Different group
 **Additional Enhancements**
 
 You can set up [vmauth](https://docs.victoriametrics.com/victoriametrics/vmauth/) for routing data to the given vminsert group depending on the needed retention.
+
+**Downsides Of This Approach**
+
+This approach requires storing the index for Victoriametrics once per retention period instead of having 1 index for all metrics, which means that this approach requires more disk space than enterprise retention filters.
```

Comment:
maybe it's better to highlight here something like "storing a separate index for each retention cluster" because you need to keep a single retention for a separate cluster